lightning/src/ln/channel.rs

```diff
@@ -7511,7 +7510,7 @@ where
 					// We really want take() here, but, again, non-mut ref :(
 					match outcome.clone() {
 						OutboundHTLCOutcome::Failure(mut reason) => {
-							hold_time(htlc.send_timestamp, now).map(|hold_time| {
+							hold_time_since(htlc.send_timestamp).map(|hold_time| {
```

Comment:
Yes the reason why I passed now is that all htlcs in this round would get the same time passed without small differences. Also seemed unnecessary to keep requesting the current time. But it matters very little to none.